addons/io_hubs_addon/debugger.py

```diff
@@ -122,8 +122,8 @@ def execute(self, context):
 
 class HubsCreateRoomOperator(bpy.types.Operator):
     bl_idname = "hubs_scene.create_room"
-    bl_label = "Create Room"
-    bl_description = "Create room"
+    bl_label = "Create a new room"
+    bl_description = "Creates a new room in the selected instance"
```

Comment:
Maybe some additional info, like this: 
```suggestion
    bl_description = "Creates a new room in the selected instance and opens it in the browser selected in the add-on preferences.  The specified room flags will be applied"
```

Reply:
Updated